src/streamlink_cli/argparser.py

```diff
@@ -1029,11 +1030,22 @@ def build_parser():
         Skip to the beginning of a live stream, or as far back as possible.
         """,
     )
+
+    transport_dash.add_argument(
+        "--dash-manifest-reload-attempts",
+        type=num(int, min=0),
+        metavar="ATTEMPTS",
+        help="""
+        How many attempts should be done to reload the DASH manifest before giving up.
```

Comment:
Wording on this seems a bit odd, "how many attempts should be done" doesn't read very well. Pretty minor nit but something like "How many attempts to reload the DASH manifest should occur before giving up." or something else.

Reply:
I've just mindlessly copied that from the HLS param help text, which was added in 2017 and hasn't been changed since.

https://github.com/streamlink/streamlink/commit/f5d862e281d430baade64976e1a6840593cbf8a9#diff-cf813af4bcbb52b3cde4c96d819cf4834df10b6f2aefbe8f9efd4afa5f08649bR608-R609

Reply:
Gotcha.